tcms/management/tests/test_baseurl.py

```diff
@@ -0,0 +1,28 @@
+from io import StringIO
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from tcms.tests.factories import SiteFactory
+
+
+class TestBaseurl(TestCase):
+    """Test manage.py baseurl command"""
+
+    def test_check_current_url(self):
+        """Test command without arguments returns current Base URL"""
```

Comment:
Prefer verbose method names instead of doc-string to explain what something does, especially when it is about testing. e.g.

`test_who|what_action_expected_result`

Reply:
I got confused by what this does below. A name like `test_without_params_returns_domain` would have been very helpful.